includes/class-transformer-base.php

```diff
@@ -0,0 +1,597 @@
+<?php
```

Comment:
I would use the `Transformer` Namespace and move them into the transformer folder.

Reply:
same with the manager

Reply:
let me know if I can help!